src/components/StudentAssignmentsTable.js

```diff
@@ -24,34 +27,50 @@ class StudentAssignmentsTable extends React.PureComponent {
 
   render() {
     const {
-      assignments, allowTestingScores, setFakeScore, resetFakeScore, removeFakeAssignment,
+      id,
+      assignments,
+      categories,
+      allowTestingScores,
+      setFakeScore,
+      resetFakeScore,
+      removeFakeAssignment,
     } = this.props;
     return (
-      <div className="uk-overflow-auto uk-margin-small-top">
-        <table className="uk-table uk-table-small uk-table-hover uk-table-divider">
-          <thead>
-            <tr>
-              <th className="uk-table-expand">Name</th>
-              <th>Score</th>
-              <th>Max</th>
-              <th>Percent</th>
-              <th>Category</th>
-            </tr>
-          </thead>
-          <tbody>
-            {assignments.map(assign => (
-              <StudentAssignmentRow
-                key={assign.id}
-                {...assign}
-                getCategoryName={this.getCategoryName}
-                allowTesting={allowTestingScores}
-                setFakeScore={setFakeScore}
-                resetFakeScore={resetFakeScore}
-                removeFakeAssignment={removeFakeAssignment}
-              />
-            ))}
-          </tbody>
-        </table>
+      <div>
+        <Modal name={`categories-table-${id}`}>
+          <h4 className="uk-text-center">Categories</h4>
+          <StudentCategoryTable categories={categories} />
+        </Modal>
+        <div className="uk-overflow-auto uk-margin-small-top">
+          <table className="uk-table uk-table-small uk-table-hover uk-table-divider">
+            <thead>
+              <tr>
+                <th className="uk-table-expand">Name</th>
+                <th>Score</th>
+                <th>Max</th>
+                <th>Percent</th>
+                <th>
+                  <a data-uk-tooltip="Click for details per category" data-uk-toggle={`target: #categories-table-${id}-modal`}>
```

Comment:
Can we fix the keyboard issue? We need to be as accessible as possible for education software.

Reply:
Agreed, but given that this is a pervasive issue throughout the code base can you open an issue instead so it can be fixed separately?

Reply:
I'll go ahead and accept this PR, let's schedule fixing accessibility for later down the road. Created #14 to reflect.